### ISSUE TITLE
Can't get OAUTH via browser popup 

### ISSUE THREAD

**Issue by JorgeMiguelGomes** [ropensci/rtweet#568]
### Problem

After revoking access to the rtweet app on one account, rtweet doesn't allow me to start the process again 

### Expected behavior

It would be expected that if no authentication is present then the popup would appear to allow access to rtweet via a twitter account 

### Reproduce the problem

``` r
## insert code here
library(rtweet)
algo <- lookup_users("jmgvostpt")

### rtweet version

<!-- run the code below and copy/paste the output -->

``` r
## copy/paste output
packageVersion("rtweet")
```
‘0.7.0’

### Session info

``` r
## copy/paste output
sessionInfo()
```
R version 4.0.5 (2021-03-31)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.0.5 tools_4.0.5    yaml_2.2.1     tinytex_0.31   xfun_0.22

**Comment by llrs** [comment]
What do you mean by "doesn't allow me to start the process again"? Which error message do you get?

I think the problem is that the token is still saved on your computer and it is not automatically removed for you. You'll need to manually remove it and then rtweet will ask you to authenticate again. 
Let me know how it goes.

**Comment by JorgeMiguelGomes** [comment]
> What do you mean by "doesn't allow me to start the process again"? Which error message do you get?
I don't  get any error message at all. 

> I think the problem is that the token is still saved on your computer and it is not automatically removed for you. You'll need to manually remove it and then rtweet will ask you to authenticate again.
> Let me know how it goes.

Sorry for the inconvenience but how do I do that?

**Comment by JorgeMiguelGomes** [comment]
> Sorry for the inconvenience but how do I do that?

I can't find the .httr-oauth file on my system after running ```sudo find . -type f -name ".httr-ouath"``` @llrs

**Comment by llrs** [comment]
Are you saying that if you do:

```r 
library(rtweet)
algo <- lookup_users("jmgvostpt")
```
You don't get any error? But you get something on algo?  
Could you post a [reprex](https://reprex.tidyverse.org/) of your code and output?

To delete the token you do not need to delete a .httr-oauth file. You should look at the ~ folder (`cd ~`) for a rtweet_token.rds file and delete it (it could also be rtweet_token1.rds). If it is not there look at your environment variables (probably you have something on .Renviron, see it with `usethis::edit_r_environ()`).

**Comment by JorgeMiguelGomes** [comment]
@llrs I didn't get any error, no and ```algo``` was empty. 
Even if I wasn't able to find ```rtweet_token.rds```  or ```rtweet_token1.rds```  via terminal, in RStudio (after installing the ```devtools``` package I was able to follow your advice and with ```usethis::edit_r_environ()``` open the file and I deleted the ```TWITTERPATH``` and saved the file. 

After restarting RStudio, and running my code, I got the popup to authorize access to RTweet. 

My knowledge level is pretty low and I'm probably not the only one than ran into this problem. It would be important if this information would be somewhere in the documentation, in my opinion. Better yet  it would be if a nice feature if the package would check if the OAUTH token is still valid and, if not, open the browser for authentication again. 

Thank you for your time and patience, and thank you for ```rtweet``` It is a truly amazing tool for Twitter analysis.
